src/components/ClusterNodes/NodeDetailsList.tsx

```diff
@@ -605,7 +586,9 @@ export default function NodeDetailsList({ isSuperAdmin, renderRefreshBar, addTab
     }
 
     return (
-        <div data-testid="cluster_name_info_page" className="node-list dc__overflow-hidden dc__border-left">
+        <div data-testid="cluster_name_info_page" className="node-list dc__overflow-hidden dc__border-left" style={{
+            height: 'calc(100vh - 84px)'
```

Comment:
Are we sure we need to add calc for height?

Reply:
yes. Otherwise the whole window scrolls.